pkg/targets/adapter/datadogtarget/adapter.go

```diff
@@ -85,6 +86,7 @@ var _ pkgadapter.Adapter = (*datadogAdapter)(nil)
 
 type datadogAdapter struct {
 	apiKey string
+	site   string
```

Comment:
nit: we could do a very slight optimization here.

Creating 2 fields at the adapter called
- `apiURL`
- `apiLogsURL`

At the `NewTarget` function we add the base to the site suffix and assign the result to those variables above.

That way we avoid an `fmt.Sprintf` for each processed event.

Reply:
It's done ✅ I was also tempted to move the `newLogsAPIRequest` and `newAPIRequest` into the struct to access it directly, let me know if that's something we'd prefer.